<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnrollStudent</class>
 <widget class="QMainWindow" name="EnrollStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enroll Student</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="RegistrationPage">
      <property name="minimumSize">
       <size>
        <width>811</width>
        <height>721</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Student Name: </string>
       </property>
      </widget>
      <widget class="QPushButton" name="backBtn">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>680</y>
         <width>371</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="shortcut">
        <string>Backspace</string>
       </property>
      </widget>
      <widget class="QPushButton" name="viewAttendanceBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>680</y>
         <width>371</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>View Attendance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>371</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="takeImagesBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>381</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Capture Images</string>
       </property>
      </widget>
      <widget class="QPushButton" name="uploadImagesBtn">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>50</y>
         <width>371</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload Images</string>
       </property>
      </widget>
      <widget class="QFrame" name="videoFrame">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>761</width>
         <height>541</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>481</width>
         <height>351</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="captureBtn">
          <property name="text">
           <string>Capture</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="quitBtn">
          <property name="text">
           <string>Quit</string>
          </property>
          <property name="shortcut">
           <string>Q</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="videoLabel">
          <property name="minimumSize">
           <size>
            <width>461</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="registerBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>640</y>
         <width>371</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QLabel" name="registerLabel">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>640</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
